packages/lambda-tiler/package.json

```diff
@@ -22,6 +22,7 @@
     "@cogeotiff/core": "^2.2.0",
     "@cogeotiff/source-aws": "^2.2.0",
     "@linzjs/geojson": "^4.16.0",
+    "node-fetch": "^2.6.1",
```

Comment:
This should be a devDependency as it is not needed at runtime.

Reply:
Good pick.

Reply:
But isn't it needed at runtime because the test is run at runtime?